src/pages/components/breadcrumbs.mdx

```diff
@@ -0,0 +1,40 @@
+---
+title: Breadcrumbs
+description: >
+  Breadcrumbs mark and communicate the current location of where a user is within the product.
+package: '@zendeskgarden/react-breadcrumbs'
+---
+
+<Usage>
+  <Use>
+    <ul>
+      <li>To inform the user of where they are in a nested navigation</li>
+      <li>To quickly navigate back to a parent page</li>
+    </ul>
+  </Use>
+</Usage>
+
+## Examples
+
+### Default
+
+Breadcrumbs work in combination with the [Anchor](/components/anchor) component for linking.
+
+import BreadcrumbsDefault from '../../examples/breadcrumbs/BreadcrumbsDefault.tsx';
+import BreadcrumbsDefaultCode from '!!raw-loader!../../examples/breadcrumbs/BreadcrumbsDefault.tsx';
+
+<CodeExample code={BreadcrumbsDefaultCode}>
+  <BreadcrumbsDefault />
+</CodeExample>
+
+## Configuration
+
+<PackageDescription data={props.data.mdx.package} />
+
```

Comment:
There should be an `##API` section here to show users how to import.

Reply:
Henry and I have been working on getting the [Configuration section up to spec](https://share.getcloudapp.com/P8uEJRd7) which is where we were planning on having that communication, but it isn't represented here. [The config branch is WIP here.](https://github.com/zendeskgarden/website/tree/rmoody/configuration)

Reply:
Got it.